test/e2e/operator_test.go

```diff
@@ -45,17 +44,11 @@ var (
 	operatorNs = "contour-operator"
 	// specNs is the spec.namespace.name of a Contour.
 	specNs = "projectcontour"
```

Comment:
maybe worth just removing this variable completely so we are not tempted to use it, and to have a test helper that generates a namespace name for each test